ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -322,7 +322,8 @@ class TensorboardLogger(BaseLogger):
 
 
     Args:
-        log_dir (str): path to the directory where to log.
+        *args: Positional arguments accepted from :class:`~tensorboardx.SummaryWriter`.
```

Comment:
```suggestion
        *args: Positional arguments accepted from :class:`~tensorboardx.SummaryWriter`, for example, `log_dir` 
            to setup path to the directory where to log.
```

Reply:
Thanks for your very fast reply :) Right at the moment, I was searching the web on how to write documentation for args and kwargs.

Reply:
Shouldn't the `log_dir` hint be added to the kwargs description?
```python
Args:
        *args: Positional arguments accepted from :class:`~tensorboardx.SummaryWriter`.
        **kwargs: Keyword arguments accepted from :class:`~tensorboardx.SummaryWriter`, for example, `log_dir` to setup path to the directory where to log.
```

Reply:
@Bibonaut yes, you are right, we can add it to kwargs